config/config.go

```diff
@@ -81,6 +81,10 @@ type configData struct {
 	GlobalLogConfig     []GlobalLogConfig     `json:"global_log_configuration"`
 }
 
+type appValidationStatus struct {
+	Error string `bson:"error"`
```

Comment:
```suggestion
	Error string `json:"error"`
```

The rdk configs go back and forth in JSON, not bson. See:

```

	// Status refers to the validations done in the APP to make sure a module is configured correctly
	Status           *appValidationStatus `json:"status"`
```